<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MalyDresiarz</class>
 <widget class="QMainWindow" name="MalyDresiarz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1161</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Start">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Wybierz tryb rozrywki:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RadioTypeLokalna">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>106</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Lokalna</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RadioTypeOnline">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>106</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Online</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Ilość:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="SpinPlayers">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>45</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="SpinComputerPlayers">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>45</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>graczy</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>graczy komputerowych</string>
      </property>
     </widget>
     <widget class="QListView" name="PlayerList">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>20</y>
        <width>581</width>
        <height>571</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="StartButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>231</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Gra">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Tura gracza:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Wybierz ruch:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="LansRadio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>106</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Lans</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="DoRobotyRadio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>106</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Do roboty</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="HaraczRadio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>106</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Haracz</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="BMWRadio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>106</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>BMW</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="IwanRadio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>106</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Iwan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MakeMove">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Wykonaj ruch</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>10</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Szacun ranking</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Twoje statystyki:</string>
      </property>
     </widget>
     <widget class="QLabel" name="SzacunStat">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Szacun:</string>
      </property>
     </widget>
     <widget class="QLabel" name="KasaStat">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Kasa:</string>
      </property>
     </widget>
     <widget class="QLabel" name="HaraczeStat">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Haracze:</string>
      </property>
     </widget>
     <widget class="QLabel" name="BMWStat">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>BMW:</string>
      </property>
     </widget>
     <widget class="QLabel" name="SzacunPred">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Szacun:</string>
      </property>
     </widget>
     <widget class="QLabel" name="KasaPred">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Kasa:</string>
      </property>
     </widget>
     <widget class="QLabel" name="BMWPred">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>210</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>BMW:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>130</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Powykonaniu ruchu:</string>
      </property>
     </widget>
     <widget class="QLabel" name="HaraczePred">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>190</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Haracze:</string>
      </property>
     </widget>
     <widget class="QLabel" name="TuraCounter">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Pozostało: x tur</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PlayerName">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Zatwierdz">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Zatwierdz</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="DresiarzRanking">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>30</y>
        <width>371</width>
        <height>611</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="npcact">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>350</y>
        <width>1</width>
        <height>1</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="npcreg">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>380</y>
        <width>1</width>
        <height>1</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1177</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
